suites/quincy/rgw/tier-1_rgw_ms_upgrade_6-0GA_to_6x_latest.yaml

```diff
@@ -25,9 +25,8 @@ tests:
                   command: bootstrap
                   service: cephadm
                   args:
-                    rhcs-version: 6.0
-                    release: rc
-                    registry-url: registry.redhat.io
+                    custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-115"
+                    custom_repo: "https://download-node-02.eng.bos.redhat.com/rel-eng/rhel-9/RHCEPH-6/RHCEPH-6.0-RHEL-9-20230313.0/compose/Tools/x86_64/os/"
```

Comment:
this custom repo will go away soon right , is this a temporary fix?

Reply:
Yes @TejasC88 , this was provided by dmfg for issue seen with recipe file being not present in http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/ for RHCS 6.0